examples/fancy_demo/main.slint

```diff
@@ -3,7 +3,7 @@
 
 global Palette := {
     property <color> window-background: #eee;
-    property <color> widget-background: #ddd;
+    property <color> background: #ddd;
```

Comment:
currious why you changed that.
The idea here was to make a difference between the widget-background and the button-background or some other background.

Reply:
That was a mistake sorry.